lib/ls.js

```diff
@@ -321,8 +321,8 @@ const getHumanOutputItem = (node, { args, color, global, long }) => {
         : ''
     ) +
     (
-      node[_invalid]
-        ? ' ' + (color ? chalk.red.bgBlack('invalid') : 'invalid')
+      node[_invalid] && !node[_dedupe]
+        ? ' ' + (color ? chalk.red.bgBlack(node[_invalid]) : node[_invalid])
```

Comment:
```suggestion
        ? ' ' + (color ? chalk.red.bgBlack(`invalid: ${node[_invalid]}`) : `invalid: ${node[_invalid]}`)
```

it feels to me like we should keep the 'invalid' in there somewhere, the red `"1" from project root` doesn't tell me that the dep is invalid, only where it's required from

Reply:
+1, color alone should never be used to convey information

Reply:
https://github.com/npm/cli/pull/3460/commits/63997f19b4dde20c201ce9b07368b1ac6f29bec3#diff-9664071d7baeb34b8d7dab2320146e15952f1ecc6316ca1cc3eeea0a3caa77f6R311-R328